jre_emul/Makefile

```diff
@@ -207,8 +207,8 @@ $(BUILD_DIR)/.jre_dist_includes: $(PUBLIC_TRANSLATED_HEADERS:%=$(GEN_OBJC_DIR)/%
     $(PUBLIC_EMULATION_HEADERS:%=$(EMULATION_CLASS_DIR)/%) | $(DIST_INCLUDE_DIR)
 	@echo copying jre_emul header files
 	@(cd $(EMULATION_CLASS_DIR) && tar cf - \
-	    $(PUBLIC_EMULATION_HEADERS:$(EMULATION_CLASS_DIR)/%=%)) \
-	    | (cd $(DIST_INCLUDE_DIR); tar xfp -)
+	    $(PUBLIC_EMULATION_HEADERS:$(EMULATION_CLASS_DIR)/%=%) \
```

Comment:
Could you remove the top-level subshell brackets, as I don't think they serve much purpose. (Make invokes each line in a new subshell already)